### ISSUE TITLE
Scripting show_contacts.py - what is the syntax?

### ISSUE THREAD

**Issue by buzuddha** [dkoes/show_contacts#1]
I can't seem to script this command. I can get the GUI interface to show contacts, but simply cannot get the syntax right! :(

```
PyMOL>show_contacts /3IXT/C/P,  /3IXT/C/H,result="contacts", cutoff=3.6, bigcutoff=4.0
  File "toplevel", line 1
    show_contacts /3IXT/C/P,  /3IXT/C/H,result="contacts", cutoff=3.6, bigcutoff=4.0
                    ^
```

I've tried scripting this chain names, chain names set to variables, nothing seems to work! What am I missing?! Sorry, clearly I've been banging my head against this wall for too long now.

Many thanks for your help!

**Comment by dkoes** [comment]
The confusion here is that the function name in python is show_contacts, but the command that is registered is `contacts`:
cmd.extend('contacts', show_contacts) #contacts to avoid clashing with cluster_mols version

`contacts /3IXT/C/P,  /3IXT/C/H,result="contacts", cutoff=3.6, bigcutoff=4.0` should work

**Comment by marksev1** [comment]
Reusing this issue for another question, why doesn't it select a "list" of residues - or make a selection by residue. It just draws the dashes and distances....So there is missing information about the residues involved? Or am I missing something?

**Comment by dkoes** [comment]
I don't understand your question.  You provide the selections so you can select whatever you want however you want.

**Comment by marksev1** [comment]
I was not implying to the input - yes I select the two selections, I was talking about the output of the script. I thought the the output of the script would be a new selection of "interfacing" residues. Yet the script provides just "lines"/"dashes" between the residues, visually indicating where the contact points would be, without actually making a selection of those "interfacing residues".

**Comment by dkoes** [comment]
Correct, that is the intended behavior.  Pull requests with improvements always appreciated.

**Comment by Stefan-Dobrev** [comment]
Damn, if I'd come here right away I would have saved a lot of time. 

Different issue - I can't get those dashes to stop showing as cylinders. The script uses the built-in distance function to draw the dashes. Settings like dash_length and dash_gap still apply, but dash_as_cylinders has no effect on the dashes generated with contacts, while it still affects dashes I create with distance.

Any ideas?
